packages/shopify-api/lib/utils/__tests__/shop-validator.test.ts

```diff
@@ -65,11 +74,16 @@ describe('sanitizeShop', () => {
     );
   });
 
-  test('returns null for invalid URLs', () => {
-    const shopify = shopifyApi(testConfig());
+  [
+    INVALID_SHOP_URL_1,
+    INVALID_SHOP_URL_2,
+    VALID_SHOPIFY_HOST_BUT_NOT_VALID_UNIFIED_ADMIN,
+  ].forEach((invalidUrl) => {
+    test('returns null for invalid URLs', () => {
```

Comment:
Slightly nitpick-y, but you can use `test.each`
```suggestion
test.each([
  INVALID_SHOP_URL_1,
  INVALID_SHOP_URL_2,
  VALID_SHOPIFY_HOST_BUT_NOT_VALID_UNIFIED_ADMIN,
])("returns null for invalid URL %s", (invalidUrl) => {
  // Test things
});
```